experiments/openebs/openebs-pool-disk-loss/openebs_pool_disk_loss_ansible_logic.yml

```diff
@@ -0,0 +1,213 @@
+    liveness_namespace: "{{ lookup('env','LIVENESS_APP_NAMESPACE') }}"
+    openebs_ns: "{{ lookup('env','OPENEBS_NAMESPACE') }}"
+    cloud_platform: "{{ lookup('env','CLOUD_PLATFORM') }}"
+    c_ns: "{{ lookup('env','CHAOS_NAMESPACE') }}"
+    disk_name: "{{ lookup('env','DISK_NAME') }}"
+    device_name: "{{ lookup('env', 'DEVICE_NAME') }}"
+    node_name: "{{ lookup('env','NODE_NAME') }}"
+    project_id: "{{ lookup('env','PROJECT_ID') }}"
+    zone_name: "{{ lookup('env','ZONE_NAME') }}"
+    auxiliary_appinfo: "{{ lookup('env','AUXILIARY_APPINFO') }}"
+
+  vars_files:
+    - /mnt/parameters.yml
+    - /experiments/openebs/openebs_components.yml
+  
+  tasks:
+    - block:
+
+      - include_tasks: /utils/runtime/create_testname.yml  
+
+      - include_tasks: openebs_pool_disk_loss_ansible_prerequisites.yml
+
+      # GENERATE EXPERIMENT RESULT NAME AND CREATE CHAOS RESULT RESOURCE
+      - block:
+          - name: Construct chaos result name (experiment_name)
+            set_fact:
+              c_experiment: "{{ lookup('env','CHAOSENGINE') }}-{{ c_experiment }}"
+            when: lookup('env','CHAOSENGINE')
+
+          - include_tasks: /utils/runtime/update_chaos_result_resource.yml
+            vars:
+              status: 'SOT'
+              namespace: "{{ a_ns }}"
+
+      # DISPLAY APP INFORMATION
+      - name: Display the app information passed via the test job
+        debug:
+          msg:
+            - "The application info is as follows:"
+            - "Namespace    : {{ a_ns }}"
+            - "Label        : {{ a_label }}"
+
+      - block: 
+        # Gcloud authentication
+        - name: Gcloud authentication
+          include_tasks: "/utils/cloud/gcp/gcloud_configure.yml"
+          when: "cloud_platform == 'GCP'"
+        
+        # AWS authentication
+        - name: AWS authentication
+          include_tasks: "/utils/cloud/aws/aws_configure.yml"
+          when: "cloud_platform == 'AWS'"
+
+      ## PRE-CHAOS DISK LIVENESS CHECK
+      - block:
+        - name: Verify that the disk is connected to node (pre)
+          include_tasks: "/utils/cloud/gcp/status_disk.yml"
+          when: "cloud_platform == 'GCP'"
+
+        - name: Verify that the disk is connected to node (pre)
+          include_tasks: "/utils/cloud/aws/status_disk.yml"
+          when: "cloud_platform == 'AWS'"
+
+        # Checking disk is attached to node
+        - debug: 
+            msg: "specified disk is attached to node"   
+          when: "inuse is defined and inuse == true"
+
+        - fail:
+            msg: "specified disk not attached to node"
+          when: "inuse is defined and inuse == false"
+        
+      - name: Verify that the AUT (Application Under Test) is running
+        include_tasks: "/utils/common/status_app_pod.yml"
+        vars:
+          app_ns: "{{ a_ns }}"
+          app_label: "{{ a_label }}"      
+          delay: 5
+          retries: 60
```

Comment:
pls, change the delay to 2 and retries to 90? as we are fixing these value for all experiments?